src/Sarif/Core/Stack.cs

```diff
@@ -124,13 +124,14 @@ public static Stack Create(string stackTrace)
                     if (!string.IsNullOrEmpty(match.Groups[2].Value))
                     {
                         string fileName = match.Groups[3].Value;
+                        Uri.TryCreate(fileName, UriKind.RelativeOrAbsolute, out Uri fileNameUri);
```

Comment:
>TryCreate [](http://example.com/codeflow?start=28&length=9)

My concern here is that if TryCreate fails, we'll create an invalid artifact location.

Now SARIF may be eliding this at serialization time! But it raises the question should we silently fail? Or do we prefer to see the exception so that we fix the code??

Reply:
yes. Reverted to keep the current behavior.